.ci/component_descriptor

```diff
@@ -1,95 +1,26 @@
 set -o nounset
 set -o pipefail
 
-echo "component-cli is required to generate the component descriptors"
-CLI_PATH="$(mktemp -d)"
-COMP_CLI=${CLI_PATH}/component-cli
-echo "Trying to installing component-cli to ${COMP_CLI}"
-OS="$(uname -o | awk '{print tolower($0)}')"
-ARCH="$(uname -m | awk '{print tolower($0)}')"
-
-if [[ ${OS} == *"linux"* ]]; then
-  OS="linux"
-fi
-if [[ ${OS} == *"darwin"* ]]; then
-  OS="darwin"
-fi
-
-if [[ ${ARCH} == *"x86_64"* ]]; then
-  ARCH="amd64"
-fi
-if [[ ${ARCH} == *"arm64"* ]]; then
-  ARCH="arm64"
-fi
-
-curl -L https://github.com/gardener/component-cli/releases/download/v0.48.0/componentcli-${OS}-${ARCH}.gz | gzip -d > ${COMP_CLI}
-chmod +x ${COMP_CLI}
+echo "create dummy component descriptor BEGIN"
 
 SOURCE_PATH="$(dirname $0)/.."
 VERSION="$(${SOURCE_PATH}/hack/get-version.sh)"
-COMMIT_SHA="$(git --git-dir ${SOURCE_PATH}/.git rev-parse HEAD)"
-
-printf "> Building components with version ${VERSION} - ${COMMIT_SHA}\n"
-
-REPO_CTX="${CURRENT_COMPONENT_REPOSITORY}"
-
-# creates a component archive for deployer
-# it expects 1 argument with
-# $1 is the name of the component
-function buildComponentArchive() {
-  COMPONENT_NAME=$1
-  CA_PATH="$(mktemp -d)"
-  printf "> Building component ${COMPONENT_NAME}\n"
-
-  COMPONENT_REFERENCES=""
-
-  if [ -f ${SOURCE_PATH}/.landscaper/${COMPONENT_NAME}/component-references.yaml ]; then
-    COMPONENT_REFERENCES="-c ${SOURCE_PATH}/.landscaper/${COMPONENT_NAME}/component-references.yaml"
-  fi
-
-  ${COMP_CLI} ca "${CA_PATH}" "${CTF_PATH}" \
-    --component-name=github.com/gardener/landscaper/${COMPONENT_NAME} \
-    --component-version=${VERSION} \
-    --repo-ctx=${REPO_CTX} \
-    -s ${SOURCE_PATH}/.landscaper/sources.yaml \
-    -r ${SOURCE_PATH}/.landscaper/${COMPONENT_NAME}/resources.yaml \
-    COMMIT_SHA=${COMMIT_SHA} \
-    VERSION=${VERSION} \
-    $COMPONENT_REFERENCES
-}
-
-buildComponentArchive "container-deployer"
-buildComponentArchive "helm-deployer"
-buildComponentArchive "manifest-deployer"
-buildComponentArchive "mock-deployer"
-
-# add landscaper component descriptor
-printf "> Create Landscaper ca archive\n"
-LS_CA_PATH="$(mktemp -d)"
-cp ${BASE_DEFINITION_PATH} "${LS_CA_PATH}/component-descriptor.yaml"
+REPO_CTX="eu.gcr.io/gardener-project/development"
+COMPONENT_DESCRIPTOR_DIR="${SOURCE_PATH}/../${COMPONENT_DESCRIPTOR_DIR}"
 
-printf "> add resources\n"
-${COMP_CLI} ca resources add ${LS_CA_PATH} \
-    VERSION=${VERSION} \
-    ${SOURCE_PATH}/.landscaper/resources.yaml
+VERSION="${VERSION}-dummy" REPO_CTX="${REPO_CTX}" COMPONENT_DESCRIPTOR_DIR="${COMPONENT_DESCRIPTOR_DIR}" python ${SOURCE_PATH}/hack/create-dummy-cd.py
 
-printf "> add component references\n"
-${COMP_CLI} ca component-references add ${LS_CA_PATH} \
-    VERSION=${VERSION} \
-    ${SOURCE_PATH}/.landscaper/component-references.yaml
+echo "create dummy component descriptor END"
 
-cat ${LS_CA_PATH}/component-descriptor.yaml
+echo "create real component descriptor BEGIN"
```

Comment:
I suppose the point here is to create _and push_ it so that some other stuff downstream (e.g. integration-tests) can rely on it. Might be useful to log that as well, maybe like
```bash
echo "Creating real component descriptor and uploading it"
```

Reply:
There is some log outputs later that log the publishing of the components.